packages/datadog-instrumentations/src/fastify.js

```diff
@@ -45,16 +46,35 @@ function wrapAddHook (addHook) {
 
     if (typeof fn !== 'function') return addHook.apply(this, arguments)
 
-    arguments[arguments.length - 1] = shimmer.wrapFunction(fn, fn => function (request, reply, done) {
+    arguments[arguments.length - 1] = shimmer.wrapFunction(fn, fn => function () {
```

Comment:
if you remove the arguments from the function signature, you potentially break any behavior in the framework, that might be checking "func.length" to detect the type of callback it is and what arguments to pass it

Reply:
yes sorry, I misunderstood it. you're right.